pkg/server/api/agent/v1/service.go

```diff
@@ -50,6 +51,8 @@ type Service struct {
 	ds  datastore.DataStore
 	ca  ca.ServerCA
 	td  spiffeid.TrustDomain
+
```

Comment:
nit: can we group this with the rest of the fields? any reason to keep it separate?